db/migrations/20240619190000_AddJobs.up.sql

```diff
@@ -0,0 +1,13 @@
+-- Add migration script here
+CREATE TABLE
+  Jobs (
```

Comment:
We did this one together! Just copied from the `cja` stuff basically

This table records one specific piece of work that we want to do